.circleci/config.yml

```diff
@@ -43,6 +43,25 @@ jobs:
           root: .
           paths:
             - ./*
+  integration-test:
+    working_directory: ~/addons-frontend
+    machine: true
+    steps:
+      - checkout
+      - run:
+          name: Install Docker Compose
+          command: |
+            set -x
+            pip install docker-compose>=1.18
+            docker-compose --version
+      - run:
+          name: Install tox
+          command: pip install tox
+      - run:
+          name: Run tests
+          command: tox -e ui-test-ci
+      - store_artifacts:
+          path: addons-server/ui-test.html
```

Comment:
Is this path correct? It doesn't seem to be given the `working_directory`

Reply:
Yes because the tests are run within the ```addons-server``` dir and the HTML report gets created there.